src/Tickets/Commerce/Order_Modifiers/Checkout/Coupons.php

```diff
@@ -99,12 +99,6 @@ public function localize_asset(): void {
 	 * @param Tribe__Template $template The template object for rendering.
 	 */
 	public function display_coupon_section( WP_Post $post, array $items, Tribe__Template $template ): void {
-		// Display the coupon section template.
-		$template->template(
-			'checkout/order-modifiers/coupons',
-			[
-				// Additional data if needed.
-			]
-		);
+		$template->template( 'checkout/order-modifiers/coupons' );
```

Comment:
Just a note that this is fine to change, but since the coupon feature is coming as a follow-up this will certainly change again.